tests/cases/fourslash/goToDefinitionProperty3.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path="./fourslash.ts"/>
+
+////interface Foo {
+////    /*2*/foo(): void
+////}
+////const foo = 1;
+////let x: Foo = {
+////    [|f/*1*/oo|]()
+////}
+
+verify.goToDefinition("1", "2");

```

Comment:
@DanielRosenwasser I think in this test no, because refers to `[|f/*1*/oo|]()` function. However, I think [goToDefinitionShorthandProperty05.ts](https://github.com/microsoft/TypeScript/pull/42924/files#diff-42c73f9324ba8a9f30837c2fd3fc6a04509bbd3c08b8777ba051bd878636daffR4-R11) should include. What do you think?